harness/harness.rb

```diff
@@ -1,51 +1,38 @@
-require 'csv'
+require 'benchmark'
 
 # Warmup iterations
-$WARMUP_ITRS = ENV.fetch('WARMUP_ITRS', 15).to_i
+WARMUP_ITRS = ENV.fetch('WARMUP_ITRS', 15).to_i
 
 # Minimum number of benchmarking iterations
-$MIN_BENCH_ITRS = 10
+MIN_BENCH_ITRS = ENV.fetch('MIN_BENCH_ITRS', 10).to_i
 
 # Minimum benchmarking time in seconds
-$MIN_BENCH_TIME = 10
+MIN_BENCH_TIME = ENV.fetch('MIN_BENCH_TIME', 10).to_i
 
-$out_csv_path = File.expand_path(ENV.fetch('OUT_CSV_PATH', 'output.csv'))
+OUT_CSV_PATH = ENV.fetch('OUT_CSV_PATH', "results-#{RUBY_ENGINE}-#{RUBY_ENGINE_VERSION}.csv")
```

Comment:
We'd need some other component here to make the default file different between ruby-head and yjit (same for mjit). Not sure what would work best. The full `RUBY_DESCRIPTION` feels a bit weird for a filename.

Reply:
Here's the RUBY_DESCRIPTION I see currently with my local ruby-yjit:

~~~
irb(main):001:0> RUBY_DESCRIPTION
=> "ruby 3.1.0dev (2021-06-10T17:53:36Z yjit_stats_ruby_de.. 06bd05df65) [x86_64-darwin20]"
~~~

My vote would be *not* to include that in the output filename.